app/views/docs/realtime.phtml

```diff
@@ -0,0 +1,341 @@
+final realtime = Realtime(client);
+final subscription = realtime.subscribe('account');
+
+subscription.stream.listen((response) {
+    // Callback will be executed on all account events.
+    print(response);
+})</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Kotlin</h3>
+        <div class="ide margin-bottom" data-lang="kotlin" data-lang-label="Kotlin SDK">
+            <pre class="line-numbers"><code class="prism language-kotlin" data-prism>val client = Client()
+val realtime = Realtime(client)
+
+realtime.subscribe("account", callback = { param ->
+    // Callback will be executed on all account events.
+    print(param.toString())
+})</code></pre>
+        </div>
+    </li>
+</ul>
+
+<h3><a href="/docs/realtime#subscribe-to-multiple-channels" id="subscribe-to-multiple-channels">Subscribe to multiple channels</a></h3>
+
+<p>You can also listen to multiple <a href="/docs/realtime#channels">channels</a> at once by passing an array of channels. This will trigger the callback for any events for all channels passed.</p>
+<p>In this example we are subscribing to the documents A and B by subscribing to the <span class="tag">documents.A</span> and <span class="tag">documents.B</span> channels.</p>
+
+<ul class="phases clear" data-ui-phases>
+    <li>
+        <h3>Web</h3>
+        <div class="ide margin-bottom" data-lang="javascript" data-lang-label="Web SDK">
+            <pre class="line-numbers"><code class="prism language-javascript" data-prism>let sdk = new Appwrite();
+
+sdk.subscribe(['documents.A', 'documents.B'], response => {
+    // Callback will be executed on changes for documents A and B.
+    console.log(response);
+});</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Flutter</h3>
+
+        <div class="ide margin-bottom" data-lang="dart" data-lang-label="Flutter SDK">
+            <pre class="line-numbers"><code class="prism language-dart" data-prism>final client = Client();
+final realtime = Realtime(client);
+final subscription = realtime.subscribe(['documents.A', 'documents.B']);
+
+subscription.stream.listen((response) {
+    // Callback will be executed on changes for documents A and B.
+    print(response);
+})</code></pre>
+        </div>
+    </li>
+    <li>
+        <h3>Kotlin</h3>
+        <div class="ide margin-bottom" data-lang="kotlin" data-lang-label="Kotlin SDK">
+            <pre class="line-numbers"><code class="prism language-kotlin" data-prism>val client = Client()
+val realtime = Realtime(client)
+
+realtime.subscribe(["documents.A", "documents.B"], callback = { param ->
+    // Callback will be executed on changes for documents A and B.
+    print(param.toString())
+})</code></pre>
+        </div>
+    </li>
+</ul>
+
+<h3><a href="/docs/realtime#unsubscribe" id="unsubscribe">Unsubscribe</a></h3>
+
+<p>If you no longer want to receive update from a subscription, you must unsubscribe your listener so that your callbacks are not longer called.</p>
```

Comment:
```suggestion
<p>If you no longer want to receive update from a subscription, you must unsubscribe your listener so that your callbacks are no longer called.</p>
```